eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -252,10 +252,9 @@
   <!-- Suppress external dependency check for jacksonwrapper overriding deserialize method -->
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck" files="com.azure.core.implementation.serializer.jsonwrapper.jacksonwrapper.JacksonDeserializer.java"/>
 
-  <!-- Suppress ConfigurationCredentialsPolicy for HttpPipelinePolicyCheck.
-   This suppression need to be deleted after AzConfig using AAD credential.
-   Issue Link: https://github.com/Azure/azure-sdk-for-java/issues/5131 -->
+  <!-- Allow ConfigurationCredentialsPolicy and SyncTokenPolicy in Implementation folder.-->
```

Comment:
Why do we allow these in the implementation folder? Since that issue is closed, I'm not sure why we have these suppressions?

Reply:
This suppression could be removed by making these classes package-private in the root package. Classes in `com.azure.data.appconfiguration` do not have to be public.

Reply:
Class 'ConfigurationCredentialsPolicy' implementing 'HttpPipelinePolicy' and should be a public class [HttpPipelinePolicy]

Reply:
Either way will suppress the one checkstyle rule.